docs/index.md

```diff
@@ -0,0 +1,32 @@
+SAML Integrator is an integrator charm for providing SAML configuration details to consumer charms which seek to authenticate using a SAML server. Configure the SAML endpoint in the integrator charm and relate consumer charms as needed.
+
+The code is licensed under the [Apache License, version 2](https://www.apache.org/licenses/LICENSE-2.0), and pull requests are accepted once you've signed a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement).
+
+For details on configuration options, see [this page](https://charmhub.io/saml-integrator/configure).
+
+## Contributing to this documentation
+
+Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/t/saml-integrator-documentation-overview/11293) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, ask a question, or make a suggestion about a potential change via the comments section.
+
+If there's a particular area of documentation that you'd like to see that's missing, please [file a bug](https://github.com/canonical/saml-integrator-operator/issues).
+
+## In this documentation
+
+| | |
+|--|--|
+|  [Tutorials](https://charmhub.io/saml-integrator/docs/tutorial-getting-started)</br>  Get started - a hands-on introduction to using the Charmed Indico operator for new users </br> |  [How-to guides](https://charmhub.io/saml-integrator/docs/how-to-contibute) </br> Step-by-step guides covering key operations and common tasks |
```

Comment:
"Charmed Indico" looks like it needs changing :)

Reply:
Fixed